en/sim_gazebo_gz/index.md

```diff
@@ -243,6 +283,12 @@ where `ARGS` is a list of environment variables including:
   The default rendering engine (OGRE 2) is not well supported on some platforms/environments.
   Specify `PX4_GZ_SIM_RENDER_ENGINE=ogre` to set the rendering engine to OGRE 1 if you have rendering issues when running PX4 on a virtual machine.
 
+- `PX4_SIM_SPEED_FACTOR`:
+  Sets the speed factor to run the simulation at [faster/slower than realtime](#run-simulation-faster-slower-than-realtime).
+
+- `PX4_GZ_FOLLOW_OFFSET_X`, `PX4_SIM_SPEED_FACTOR_Y`, `PX4_SIM_SPEED_FACTOR_Z`:
+  Set the relative offset of the follow camera to the vehicle.
```

Comment:
@dakejahl Is this correct? Is it just an initial value, or does it fix the camera position forever?

I don't see anything else in https://github.com/PX4/PX4-user_guide/pull/3628 that needs docs adjustment right?

Reply:
No but we'll need docs for the new gazebo plugins. 
https://github.com/PX4/PX4-Autopilot/pull/24441

Might be easier for us to chat sometime on discord or on a call so I can give you more information about it. The plugins are still under development so I kinda want to wait a little while longer before we make docs for it.

Reply:
Thanks @dakejahl . I've created https://github.com/PX4/PX4-Autopilot/issues/24497 and assigned both of us to this to track. Let me know when you're ready to talk about it.